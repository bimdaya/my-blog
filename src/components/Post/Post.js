import React from 'react';

import postStyles from './Post.module.css';

const Post = (post) => {
  return (
    <div>
      <h1>{post.title}</h1>
      <div className="max-w-2xl mx-auto">
        <div
          className={postStyles['markdown']}
          dangerouslySetInnerHTML={{ __html: post.content }}
        />
      </div>
    </div>
  );
};

export default Post;
