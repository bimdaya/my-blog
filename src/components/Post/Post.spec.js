import React from 'react';
import { render } from '@testing-library/react';
import Post from './Post';

describe('Post', () => {
  it('renders', () => {
    const post = { title: 'Test title', content: 'Test content' };
    const { container } = render(<Post post={post} />);

    expect(container.firstChild).toMatchSnapshot();
  });
});
