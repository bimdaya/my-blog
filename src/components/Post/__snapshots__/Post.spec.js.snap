// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Post renders 1`] = `
<div>
  <h1 />
  <div
    class="max-w-2xl mx-auto"
  >
    <div
      class="markdown"
    />
  </div>
</div>
`;
